jekyll/_cci2/style/using-code-samples.adoc

```diff
@@ -1,12 +1,18 @@
 ---
 toc: false
 ---
-
 = Using code samples
 :page-layout: classic-docs
 :icons: font
 :toc-title:
 
-* When using example code snippets, be mindful that people are likely to copy paste them. Always point out, just before the snippet, which, if any, items or values need to be replaced with the reader’s own data – including whether they should replace the surrounding `< >` or `“ “`.
+
+* Provide code examples when possible and if they will help with clarity and reproducibility for the reader.
```

Comment:
This sentence just sounds a little awkward to me, specifically from "...when possible and if..."